app/routes/my-projects/to-review.js

```diff
@@ -1,4 +1,8 @@
 import Route from '@ember/routing/route';
 
 export default class MyProjectsToReviewRoute extends Route {
+  async model() {
+    const projects = this.store.findAll('project');
+    return projects;
```

Comment:
tiny nit but you can just say `return this.store.findAll('project');` for brevity.

Reply:
yep